### PR TITLE
feat: job

### PR DESCRIPTION
**Pull request by zac-li**

### CODE REVIEW

File: lcserve/__main__.py [hunk 1]
```diff
@@ -1029,6 +1049,58 @@ async def remove(app_id):
...
+@syncify
+async def get_job(job_name, flow_id):
+    await get_job_on_jcloud(job_name, flow_id)
+
+
+@job.command('create', help='Create a job.')
+@click.argument('flow-id')
+@click.argument('job-name')
+@click.option('--param', multiple=True, help='Parameter in the form key=value.')
+def create_job(flow_id, job_name, param):
```
Comment (deepankarm) [inline lcserve/__main__.py:1078]: `params` not `param`?

File: lcserve/__main__.py [hunk 2]
```diff
@@ -1029,6 +1049,58 @@ async def remove(app_id):
...
+    from rich import print
+
+    token = Auth.get_auth_token()
+    if not token:
+        print('You are not logged in, please login using [b]jcloud login[/b] first.')
+
+    params = dict(x.split('=') for x in param)
```
Comment (deepankarm) [inline lcserve/__main__.py:1085]: What does this do? Can you please add a description?

**Review by deepankarm — CHANGES_REQUESTED** [review]
Looks good overall. Do you plan to add an integration test for an end-to-end use case?

File: lcserve/__main__.py (hunk 1 above)
Comment (zac-li) [reply lcserve/__main__.py]: yea fixed

File: lcserve/__main__.py (hunk 2 above)
Comment (zac-li) [reply lcserve/__main__.py]: removed, I changed to this kind of input:

```
python -m lcserve job create langchain-21a817c931 myjob --params doc_name state_of_the_union.txt --params question 'What did the president say about Ketanji Brown Jackson'
```

**Comment by zac-li** [comment]
@deepankarm added a simple integration test and some docs

**Pull request merged**